Source/WebKit/WebProcess/WebPage/Cocoa/WebPageCocoa.mm

```diff
@@ -1152,7 +1152,10 @@
         WebCore::TextIndicatorOption::IncludeSnapshotOfAllVisibleContentWithoutSelection,
         WebCore::TextIndicatorOption::ExpandClipBeyondVisibleRect,
         WebCore::TextIndicatorOption::SkipReplacedContent,
-        WebCore::TextIndicatorOption::RespectTextColor
+        WebCore::TextIndicatorOption::RespectTextColor,
+#if PLATFORM(VISION)
```

Comment:
this option also should be added in

```
std::optional<TextIndicatorData> textPreviewDataForRange(Document&, const SimpleRange& scope, const CharacterRange& range)
```

Reply:
I did add it to textPreviewDataForRange in Source/WebCore/page/IntelligenceTextEffectsSupport.cpp. I didn't see another function with that name, is that the one you are referring to?

Reply:
ah, sorry, I missed that!